src/pytest_cov/plugin.py

```diff
@@ -59,6 +59,9 @@ def pytest_configure(config):
     """Activate coverage plugin if appropriate."""
     if config.getvalue('cov_source'):
         if not config.pluginmanager.hasplugin('_cov'):
+            if not config.option.cov_report:
```

Comment:
I tried to remove those lines and it seemed to work fine without 'em. Did I miss something?